src/Components/SongDisplay/SongDisplay.test.js

```diff
@@ -0,0 +1,127 @@
+import { render, screen, act, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import SongDisplay from "./SongDisplay";
+
+const contextValue = {
+  playlistName: "playlist name",
+  tracks: [
+    {
+      name: "MOCKSHOW",
+      image: "https://fakeimage",
+      preview: "https://fakepreview",
+    },
+  ],
+};
+
+
+// ************************************************************************************
+// The issue I am running into is related to finding the poster. It seems that only the first test can find the poster element and then any subseqent tests that are supposed to find the poster, are unable to. If run only one test suite and skip the rest, then the first test in that suite passes. So the issue seems to do something with finding the poster multiple times across these different tests.
+// ************************************************************************************
+
+
+//this test suite only passes if these two tests are written in this order. Not sure why the order would be impacting the abilty for the tests to pass.
+describe("AlbumCover", () => {
+  test("poster is initially rendered with blurred class", async () => {
+    act(() => {
+      render(<SongDisplay {...contextValue} />);
```

Comment:
I haven't worked with useContext before but reading this blog it looks like you might need to render it inside it's context for the test? not sure 
https://polvara.me/posts/mocking-context-with-react-testing-library/